hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -949,6 +950,7 @@ protected void configureRecon() throws IOException {
 
       conf.set(OZONE_RECON_HTTP_ADDRESS_KEY, "0.0.0.0:0");
       conf.set(OZONE_RECON_DATANODE_ADDRESS_KEY, "0.0.0.0:0");
+      conf.set(OZONE_RECON_TASK_SAFEMODE_WAIT_THRESHOLD, "500");
```

Comment:
Should be `300`?
```suggestion
      conf.set(OZONE_RECON_TASK_SAFEMODE_WAIT_THRESHOLD, "300");
```

Reply:
Thanks for pointing out @smengcl . This has been fixed.